config.py

```diff
@@ -33,6 +34,17 @@ def __init__(self):
         self.create_xray_when_sending.setChecked(prefs['create_xray_when_sending'])
         self.l.addWidget(self.create_xray_when_sending)
 
+        self.expand_aliases = QCheckBox('Auto associate split aliases [?]')
+        self.expand_aliases.setChecked(prefs['expand_aliases'])
+        self.expand_aliases.setWhatsThis('When enabled, this will split aliases up further.\n\nExample: If a character on goodreads named "Vin" has a Goodreads alias of "Valette Renoux", this option will add "Valette" and "Renoux" as aliases. You may not want this in cases such as "Timothy Cratchit" who has a Goodreads alias of "Tiny Tim". Having this feature on would add "Tiny", and "Tim" as aliases which is not valid.')
```

Comment:
Why do we need a very similar (identical?) string twice?  If we need both tooltip and whatsthis, can one be derived from the other?  Or have one string we assign to both?